ddtrace/tracer.py

```diff
@@ -7,6 +7,7 @@
 from .writer import AgentWriter
 from .span import Span
 from . import compat
+from os import getpid
```

Comment:
Here are the results of the benchmark I added:

```
## tracer.trace() wrapper benchmark: 10000 loops ##
No handlers could be found for logger "ddtrace.span"
- staticmethod execution time: 0.757570
- classmethod execution time: 0.801485
- method execution time: 0.812913
## tracer.trace() benchmark: 10000 loops ##
- trace execution time: 1.352698
## getpid wrapper benchmark: 10000 loops ##
- getpid execution time: 0.001101
```
getpid amounts to 0.1% of staticmethod execution time.